.gitignore

```diff
@@ -1,2 +1,3 @@
 build
 *.pyc
+docs
```

Comment:
Not sure why this is here? In fact, I'm not sure how the `docs` directory still exists with this in place?! The way that the gitignore file should work is any lines in there that match paths, should be ignored. I.e. `build` ensures that none of the build artefacts (e.g. `product.iso`) are saved in the version history. Same with `*.pyc` - it will ensure that no `.pyc` files should be included in the version control. So having `docs` in there too should mean that the `docs` dir (or if there was a file named `docs` instead) should not be included?! But it's clearly included down here?! Regardless of why that still exists, we should not include it here.

Reply:
Removed per PR review.